### ISSUE TITLE
Nav: High Contrast Mode issues (Focused Item not visible as well as Selected Nav item not able to differentiate) 

### ISSUE THREAD

**Issue by v-jashr** [microsoft/fluentui#13162]
[Accessibility - Nav Component ]: High Contrast Mode issues (Focus Item not visible as well as Selected Nav item not able to differentiate) 

Browser
Edge Chromium (any)
Expected Result
Focus Should be visible in high Contrast mode on NAV item.
We should able to differentiate the Selected NAV item from other in High Contrast mode 

Actual Result
Focus is not visible in high Contrast mode on NAV item.
Selected NAV item is not Highlighted from in High Contrast mode 

![image](https://user-images.githubusercontent.com/50479173/81936507-0eca3f00-9610-11ea-928a-19b6fc0ac8d3.png)

![image](https://user-images.githubusercontent.com/50479173/81936473-01ad5000-9610-11ea-96d0-479ccb488004.png)

**Comment by aneeshack4** [comment]
@v-jashr Thanks for reaching out to us. This might just be an issue with Chromium itself. I wonder what it used to look like on Edge pre-Chromium - would be good to know what the expected design here is in both high contrast modes. Unless there's something wrong with the high contrast selectors themselves. @ecraig12345 could you dig further and look into this?

**Comment by ecraig12345** [comment]
In the future @v-jashr please fully fill out our Accessibility Issue template (including detailed repro steps) instead of blanking it.

Regarding the actual issue, it looks like we did have some focus styling specific to high contrast, but it may be broken due to #11742--@xugao does this look like an instance of that issue to you?
https://github.com/microsoft/fluentui/blob/7a613256ef792c037702a47fa07713cecacbc755/packages/office-ui-fabric-react/src/components/Nav/Nav.styles.ts#L117-L124

We also have this styling for the selected item, but no high contrast-specific styling (not sure if/when that should be needed).
https://github.com/microsoft/fluentui/blob/7a613256ef792c037702a47fa07713cecacbc755/packages/office-ui-fabric-react/src/components/Nav/Nav.styles.ts#L139-L140

**Comment by ecraig12345** [comment]
Confirmed this styling is correct in IE, which means it's an instance of the Chromium Edge high contrast issues.
